bonfire/config.py

```diff
@@ -22,6 +22,7 @@
 DEFAULT_CONFIGMAPS_DIR = get_config_path().joinpath("configmaps")
 
 DEFAULT_NAMESPACE_POOL = "default"
+AI_SPECIFIC_POOLS = ["ai-development"]
```

Comment:
Are we doing anything with this var anymore?

Reply:
I think previously you were using this to check if the user is trying to make a reservation in the 'ai-development' pool but they have not specified a team. I think that is a good check to keep around. In `_check_and_reserve_namespace`, you could check if the requested pool is in `conf.AI_SPECIFIC_POOLS` (perhaps rename this to `POOLS_REQUIRING_TEAM`) and if team is not defined, throw an error.